example_app/index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
```

Comment:
The current `docs/index.html` is available at https://rerun-io.github.io/ewebsock/ - I'd like to keep that.

Maybe you can copy the setup from https://github.com/emilk/eframe_template ?

Reply:
i added some files from eframe